src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/CreateCiagInductionRequestMapper.kt

```diff
@@ -20,11 +19,11 @@ class CreateCiagInductionRequestMapper(
     return CreateInductionDto(
       prisonNumber = prisonNumber,
       workOnRelease = workOnReleaseMapper.toCreateWorkOnReleaseDto(request),
-      previousQualifications = qualificationsMapper.toCreatePreviousQualificationsDto(
+      previousQualifications = qualificationsAndTrainingMapper.toCreatePreviousQualificationsDto(
         request = request.qualificationsAndTraining,
         prisonId = prisonId,
       ),
-      previousTraining = previousTrainingMapper.toCreatePreviousTrainingDto(
+      previousTraining = qualificationsAndTrainingMapper.toCreatePreviousTrainingDto(
```

Comment:
It occurred to me that I was never going to be able to convert from two separate objects (`PreviousQualifications` and `PreviousTraining`) to one (`EducationAndQualificationResponse`) using two separate mappers, so I've combined `qualificationsMapper` and `previousTrainingMapper` into one `qualificationsAndTrainingMapper`.